securedrop/bin/run-test

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
+# shellcheck disable=SC1090
+
 set -euo pipefail
-# shellcheck source=./dev-deps
-. "${BASH_SOURCE%/*}/dev-deps"
+
+source "${BASH_SOURCE%/*}/dev-deps"
+
+run_xvfb &
+run_redis &
+urandom
+run_sass
+maybe_create_config_py
 
 if [ -n "${CIRCLE_BRANCH:-}" ] ; then
     touch tests/log/firefox.log
     function finish {
         cp tests/log/firefox.log /tmp/test-results/logs/
-        cp /tmp/securedrop*log /tmp/test-results/logs/
         bash <(curl -s https://codecov.io/bash)
```

Comment:
I wonder if this is broken because codecov isn't searching in `/tmp/test-results` where the coverage reports are -- we could add a `cd /tmp/test-results` prior to running `bash <(curl -s https://codecov.io/bash)`

Reply:
It occurs to me that it could be because codecov keeps the **absolute** path of the sources instead of the relative path. And since the working directory of the Docker container changed, that could create the confusion.